awscli/customizations/datapipeline/__init__.py

```diff
@@ -400,9 +399,18 @@ def _list_runs(self, parsed_args, parsed_globals):
         formatter(self.NAME, converted)
 
     def _describe_objects(self, pipeline_id, object_ids):
-        parsed = self.client.describe_objects(
-            pipelineId=pipeline_id, objectIds=object_ids)
-        return parsed
+        # DescribeObjects will only accept 100 objectIds at a time,
+        # so we need to break up the list passed in into chunks that are at
+        # most that size. We then aggregate the results to return.
+        objects = []
+        max_items_per_describe = 100
```

Comment:
It's normally customary to put constants at module scope with ``ALL_CAPS_NAMES``.